test/e2e/scheduler_predicates.go

```diff
@@ -136,30 +136,17 @@ func cleanupPods(c *client.Client, ns string) {
 }
 
 var _ = Describe("SchedulerPredicates", func() {
+	framework := Framework{BaseName: "sched-pred"}
```

Comment:
You probably want to extend timeout for namespace deletion. This test saturates the cluster by creating a lot of pods, and thus events. I added it recently: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework.go#L41